docs/APIs/tasks/events.md

```diff
@@ -144,4 +144,4 @@ We used `InterruptIn` for the example above, but the same kind of code can be us
 
 ## Where to go from here
 
-We have discussed only a small part of how event queues work in mbed OS. The `EventQueue` and `Event` classes in the `mbed-events` library offer a lot of features that this document does not cover, including calling functions with arguments, queueing functions to be called after a delay or queueing functions to be called periodically. The [README of the mbed-events library](https://github.com/ARMmbed/mbed-os/blob/master/events/README.md) shows more ways to use events and event queues. For more details about how the events library is implemented, review [this file](https://github.com/ARMmbed/mbed-os/blob/master/events/equeue/README.md).
+We have discussed only a small part of how event queues work in mbed OS. The `EventQueue` and `Event` classes in the `mbed-events` library offer a lot of features that this document does not cover, including calling functions with arguments, queueing functions to be called after a delay or queueing functions to be called periodically. The [README of the mbed-events library](https://github.com/ARMmbed/mbed-os/blob/master/events/README.md) shows more ways to use events and event queues. To see the implementation of the events library, review [the equeue library](https://github.com/ARMmbed/mbed-os/tree/master/events/equeue).
```

Comment:
Ah sorry. That link will work ok, at this point it's basically just pointing to the source code.